packages/avatars/demo/avatar.stories.mdx

```diff
@@ -24,13 +24,25 @@ import { AVATAR_TYPE as TYPE } from './stories/data';
       badge: { control: 'number' },
```

Comment:
According to the docs and types, `badge` accepts a `ReactNode`. Can we illustrate that this would accept a `number`, `string`, or a `ReactElement` of somekind? This might involve a dropdown with `number`, `string`, `element` with sample values perhaps.